spec/models/form_spec.rb

```diff
@@ -67,6 +67,18 @@
       expect(form.errors[:name]).to include("can't be blank")
     end
 
+    it "requires available languages" do
+      form.available_languages = ""
+      expect(form).to be_invalid
+      expect(form.errors[:available_languages]).to include("can't be blank")
+    end
+
+    it "disallows invalid available languages" do
+      form.available_languages = %w[cn jp]
+      expect(form).to be_invalid
+      expect(form.errors[:available_languages]).to include("is not included in the list")
+    end
+
```

Comment:
I think we need a test case for the valid case where we set avalable_languages to ["en", "cy"]